gems/smithy/lib/smithy/templates/client/endpoint_parameters.erb

```diff
@@ -4,7 +4,17 @@
 
 module <%= namespace %>
   # Endpoint parameters used to resolve endpoints per request.
-  <%= documentation %>
+<% parameters.each do |param| -%>
+  # @!attribute <%= param.name %>
+<% if param.documentation -%>
+<% param.documentation.split("\n").each do |line| -%>
```

Comment:
I was suggesting that param.documentation should return an array of strings, then you can just iterate it here, without having to check if param.documentation exists?